server/lib/nl/config_builder/builder.py

```diff
@@ -107,6 +108,16 @@ def build(state: PopulateState, config: Config) -> BuilderResult:
     # level.
     if user_message:
       cspec.info_message = ''
+
+    if cspec.chart_type == ChartType.ANSWER_WITH_ENTITY_OVERVIEW:
+      if len(cspec.props) > 1 or len(cspec.entities) > 1:
+        # TODO: handle this case
+        continue
```

Comment:
nit: update the counter with unsupported (just in case we see it while debugging)

Reply:
done